universal-application-tool-0.0.1/app/auth/SamlCiviFormProfileAdapter.java

```diff
@@ -0,0 +1,172 @@
+package auth;
+
+import com.google.common.base.Preconditions;
+import com.google.common.base.Strings;
+import com.google.common.collect.ImmutableSet;
+import java.util.ArrayList;
+import java.util.Locale;
+import java.util.Optional;
+import javax.inject.Provider;
+import models.Account;
+import models.Applicant;
+import org.pac4j.core.context.WebContext;
+import org.pac4j.core.context.session.SessionStore;
+import org.pac4j.core.credentials.Credentials;
+import org.pac4j.core.profile.UserProfile;
+import org.pac4j.core.profile.creator.AuthenticatorProfileCreator;
+import org.pac4j.core.profile.definition.CommonProfileDefinition;
+import org.pac4j.saml.client.SAML2Client;
+import org.pac4j.saml.config.SAML2Configuration;
+import org.pac4j.saml.profile.SAML2Profile;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import repository.UserRepository;
+
+public class SamlCiviFormProfileAdapter extends AuthenticatorProfileCreator {
+
+  private static Logger LOG = LoggerFactory.getLogger(SamlCiviFormProfileAdapter.class);
```

Comment:
LOG -> logger

Only deeply immutable things should be all caps

https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names

Reply:
fixed